packages/fuel-indexer/src/service.rs

```diff
@@ -286,3 +299,23 @@ async fn create_service_task(
         }
     }
 }
+
+async fn get_start_block(
+    conn: &mut IndexerConnection,
+    manifest: &Manifest,
+) -> Option<u64> {
```

Comment:
In all cases, `get_start_block` seems to return a `u64`? So maybe we can just make `u64` the return type as opposed to `Option<u64>`?